packages/sign-client/src/controllers/engine.ts

```diff
@@ -1470,6 +1477,7 @@ export class Engine extends IEngine {
       const verifyContext = await this.getVerifyContext({
         attestationId: attestation,
         hash: hashMessage(JSON.stringify(payload)),
+        encryptedId,
```

Comment:
@ganchoradkov isn't one of these redundant with the other?

Reply:
nope because 
- `attestationId` = the jwt
- `hash` = the unecnrypted payload used for verify v1
- `encryptedId` = hash of the encrypted message used for verify v2

`encryptedId` is used for the check against `jwt.payload.id`